graphql-language/src/ast.rs

```diff
@@ -283,16 +282,11 @@ pub enum Type<'a> {
 }
 
 /// NamedType : Name
-pub struct NamedType<'a> {
```

Comment:
Why would we? There is no visible gain, and the visible drawback that we have to add another parsing method for something that will have the exact same value as the value it contains... If you can give me a realistic reason as to why it should be this way, I'll do it.